TombRaider1996/EmulatorData.cs

```diff
@@ -0,0 +1,51 @@
+﻿using LiveSplit.ComponentUtil;
+using System;
+
+namespace TR1
+{
+    /// <summary>
+    /// The emulator's watched memory addresses.
+    /// </summary>
+    internal class EmulatorData
+    {
+        /// <summary>
+        /// Serial of the running game.
+        /// For all TR1 version serials see: http://redump.org/discs/system/psx/letter/t/
+        /// Used to differentiate game versions.
+        /// </summary>
+        public StringWatcher serial { get; }
+
+        /// <summary>
+        /// Monitors a region of the memory which contains the names of the files contained in the root directory of the currently running game.
+        /// Used to tell apart versions US_1_0 and US_final as they have the same serial.
+        /// </summary>
+        public StringWatcher rootDirectoryContents { get; }
+
+        /// <summary>
+        /// Initializes <see cref="EmulatorData"/> based on <paramref name="version"/>.
+        /// </summary>
+        /// <param name="version"></param>
```

Comment:
Boilerplate doc details. Can add whatever info desired.
```suggestion
        /// <param name="version">Process version.</param>
```

Reply:
Done.